src/server/cmd/pachctl/main.go

```diff
@@ -7,12 +7,14 @@ import (
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
 	"github.com/pachyderm/pachyderm/v2/src/internal/tracing"
 	"github.com/pachyderm/pachyderm/v2/src/server/cmd/pachctl/cmd"
 	"github.com/spf13/pflag"
 )
 
 func main() {
+	ctx := pctx.Background("pachctl")
```

Comment:
This needs to happen after InitPachctlLogger.

Reply:
It’s a bit surprising to combine global and dynamic state — would it be preferable to initialize the logger inside `pctx.Background`, or return a pachtl-specific context from `pctx.InitPachctlLogger`, or have `pachctl` mutate its contextual logger?

Regardless, update to move it afterwards.